src/gsi_ncdiag/gsi_ncdiag.py

```diff
@@ -926,8 +928,10 @@ def toIODAobs(self, OutDir, TotalBias=False, clobber=True, platforms=None):
 
                 if v == 'bend':
                     # sort record_number
-                    record_number = self.var('record_number')[idx]
-                    id_recordnum_sort = sorted(range(len(record_number)), key=record_number.__getitem__)
+                    idx = range(self.nobs)
```

Comment:
I removed the sorting as `record_number` is not present in the ncDiag `conv_gps` file

Reply:
This might be something that we added for validation @XuanliLi-NOAA do you know? If so, perhaps we add a (pseudo code) 'if variable in file, sort, otherwise don't'

Reply:
In conv_gps file, is there a variable called "Observation_Subtype"? If so, you can use it for "record_number". We tried to comment this line out before, but it didn't work because the sorting of record_number not only used in toGeovals but also in toIODAobs (about 150 lines after the toGeovals).

Reply:
thank you @XuanliLi-NOAA that worked!
I thought the `Observation_Subtype` was receiver ID but indeed it looks like the number to track the occultation profile!

here's an example from `diag_conv_gps_ges.2022021600.nc4`:
```
(Pdb) np.unique(self.var('Observation_Subtype'))
array([   1,    2,    3, ..., 2237, 2238, 2239], dtype=int32)
```